Doc/library/dis.rst

```diff
@@ -1114,8 +1114,13 @@ All of the following opcodes use their arguments.
 .. opcode:: RAISE_VARARGS (argc)
 
    Raises an exception. *argc* indicates the number of arguments to the raise
-   statement, ranging from 0 to 3.  The handler will find the traceback as TOS2,
-   the parameter as TOS1, and the exception as TOS.
+   statement, ranging from 0 to 2. These parameter counts correspond to the 3
+   different forms of the ``raise`` statement:
```

Comment:
I find the suggested wording confusing.
What about simply using:
``Raises an exception using one of the 3 forms of the raise statement, depending on the value of *argc*:``
or
``Raises an exception.  *argc* indicates which of the 3 forms of the raise statement to use:``
followed by the list

Reply:
I prefer the second one. It's more concise than the first suggest and explains better what `argc` does.

Reply:
@csabella @berkerpeksag @ncoghlan what do you think about Ezio's suggest?

Reply:
Ezio's first suggestion sounds good to me.